### DISCUSSION TITLE
类型“RuleObject”上不存在属性“fullField”

### DISCUSSION THREAD

**Discussion by szym0807** [vueComponent/ant-design-vue#7702] [General]
![image](https://github.com/vueComponent/ant-design-vue/assets/99778162/1fe83687-fda8-4ee2-9770-1c3db6452c8b)
这个rule的类型应该如何处理，ts显示没有fullField这个字段

**Comment by szym0807** [comment]
![image](https://github.com/vueComponent/ant-design-vue/assets/99778162/bfa216d3-8978-49a6-82b9-03d71ee751b0)

**Comment by szym0807** [comment]
help me

**Reply by daiwanxing** [reply]
`RuleObject` 接口并没有声明 `fullField` 这个属性，这是你自定义的吧？

或者你可以：
```ts
type ExtendRuleObject = RuleObject & { fullField: string }
```
